test/unit/reducers/channels.spec.js

```diff
@@ -8,6 +10,16 @@ import channelsReducer, {
   OPENING_FAILURE
 } from 'reducers/channels'
 
+jest.mock('electron', () => {
```

Comment:
This shouldn't be needed any longer as we've added a manual mock for electron here: https://github.com/LN-Zap/zap-desktop/blob/master/test/unit/__mocks__/electron.js

Reply:
👍